packages/components/fields/multiline-text-field/src/multiline-text-field.tsx

```diff
@@ -0,0 +1,222 @@
+
+const sequentialId = createSequentialId('multiline-text-field-');
+
+const hasErrors = (errors: { [s: string]: unknown } | ArrayLike<unknown>) =>
+  errors && Object.values(errors).some(Boolean);
+
+type TErrorRenderer = (key: string, error?: boolean) => ReactNode;
+
+export type TMultiTextFieldProps = {
+  // FieldLabel
+  /**
+   * Used as HTML id property. An id is auto-generated when it is not specified.
+   */
+  id: string;
+  /**
+   * Horizontal size limit of the input fields.
+   */
+  horizontalConstraint?:
+    | 6
+    | 7
+    | 8
+    | 9
+    | 10
+    | 11
+    | 12
+    | 13
+    | 14
+    | 15
+    | 16
+    | 'scale'
+    | 'auto';
+  /**
+   * A map of errors. Error messages for known errors are rendered automatically.
+   * <br />
+   * Unknown errors will be forwarded to `renderError`
+   */
+  errors: {
+    missing: boolean;
+  };
+  /**
+   * Called with custom errors. This function can return a message which will be wrapped in an ErrorMessage. It can also return null to show no error.
+   * <br />
+   * Signature: `(key, error) => React.node`
+   */
+  renderError?: TErrorRenderer;
+  /**
+   * Indicates if the value is required. Shows an the "required asterisk" if so.
+   */
+  isRequired: boolean;
+  /**
+   * Indicates whether the field was touched. Errors will only be shown when the field was touched.
+   */
+  touched: boolean;
+
+  // TextInput
+  /**
+   * Used as HTML `autocomplete` property
+   */
+  autoComplete: string;
+  /**
+   * Used as HTML name of the input component. property
+   */
+  name: string;
+  /**
+   * Value of the input component.
+   */
+  value: string;
+  /**
+   * Called with an event containing the new value. Required when input is not read only. Parent should pass it back as value.
+   * <br />
+   * Signature: `(event) => void`
+   */
+  onChange?: ChangeEventHandler;
+  /**
+   * Called when input is blurred
+   * <br />
+   * Signature: `(event) => void`
+   */
+  onBlur?: FocusEventHandler;
+  /**
+   * Called when input is focused
+   * <br />
+   * Signature: `(event) => void`
+   */
+  onFocus?: FocusEventHandler;
+  /**
+   * Focus the input on initial render
+   */
+  isAutofocussed?: boolean;
+  /**
+   * Expands multiline text input initially
+   */
+  defaultExpandMultilineText: boolean;
+  /**
+   * Indicates that the input cannot be modified (e.g not authorized, or changes currently saving).
+   */
+  isDisabled?: boolean;
+  /**
+   * Indicates that the field is displaying read-only content
+   */
+  isReadOnly?: boolean;
+  /**
+   * Placeholder text for the input
+   */
+  placeholder?: string;
+
+  // LabelField
+  /**
+   * Title of the label
+   */
+  title?: string;
+  /**
+   * 	Hint for the label. Provides a supplementary but important information regarding the behaviour of the input (e.g warn about uniqueness of a field, when it can only be set once), whereas `description` can describe it in more depth. Can also receive a `hintIcon`.
+   */
+  hint?: string;
```

Comment:
```suggestion
  hint: string | ReactNode
```
Also, show a warning. Since required was used.